src/main/java/br/codenation/hospital/manager/model/Hospital.java

```diff
@@ -51,5 +56,53 @@ public Hospital(
     this.availableBeds = availableBeds;
     this.latitude = latitude;
     this.longitude = longitude;
+    this.patients = new HashMap<>();
+    this.stock = new HashMap<>();
+  }
+
+  public Patient addNewPatient(Patient patient) {
```

Comment:
This function can be replace for this:
```suggestion
  public Patient addNewPatient(Patient patient) {
         if (patients.containsKey(patient.getId())) {
      return null;
    }
    
    return patients.putIfAbsent(patient.getId(), patient);
  }
```

Reply:
Makes the code cleaner